js/modules/loki_app_dot_net_api.js

```diff
@@ -62,6 +62,33 @@ class LokiAppDotNetAPI extends EventEmitter {
     return thisServer;
   }
 
+  static async getServerTime() {
+    const url = `${window.getDefaultFileServer()}/loki/v1/time`;
+    let timestamp;
+
+    try {
+      const res = await nodeFetch(url);
+      if (res.ok) {
+        timestamp = await res.json();
+      }
+    } catch (e) {
+      timestamp = NaN;
```

Comment:
We won't need to set `timestamp` to `NaN` here if we set it to default above. Might be better to add a log here?